code/modules/projectiles/guns/smgs.dm

```diff
@@ -353,3 +353,83 @@
 	scatter_unwielded = 4
 	aim_slowdown = 0.15
 	wield_delay = 0.2 SECONDS
+
+
+//-------------------------------------------------------
+//Vector, based on KRISS vector 45ACP.
+/obj/item/weapon/gun/smg/vector
```

Comment:
```suggestion
/obj/item/weapon/gun/smg/vector
```
We (quite literally) have a standardized naming form for this. Probably heavy_smg or something.

should be standard_heavysmg or whatever hits your fancy as a term